WordPress/Classes/ViewRelated/NUX/LoginEmailViewController.swift

```diff
@@ -145,6 +151,10 @@ class LoginEmailViewController: LoginViewController, NUXKeyboardResponder {
         googleLoginButton = button
     }
 
+    func configureGoogleButton() {
```

Comment:
Moving the conversation from #8770 to here:

> However, the font needs to be set every time the `traitCollection` changes. Are you suggesting adding one or more methods to `WPStyleGuide+Login` so that code in this class would look similar to this?
`
func configureGoogleButton() {
    WPStyleGuide.styleGoogleButton(googleLoginButton)
}
`

Good question. I didn't have a specific implementation in mind, but my desire is to not have to add methods to `LoginEmailViewController.swift` at all. Ideally the buttons can manage their own font size after the dynamic type preference has changed.

For example, it could be done with a [private subclass of UIButton, like this](https://github.com/wordpress-mobile/WordPress-iOS/commit/352863caf70d6f419ad3b3a9286fc54f0af0a6d2). You can see how those three buttons (the terms button on the signup popover as well) get the behaviour without having to add any code outside `WPStyleGuide+Login.swift`. 

Fwiw, don't be afraid to change things that might have wider implications, eg if you want to edit `NUXButton`, feel free! It's yours too—go ahead break it 😀

Reply:
Thanks @nheagy I see your point but I would like to avoid subclassing again. 

Yes, the buttons should manage themselves, and there are APIs for that, I just couldn't make them work. However, I had totally missed that private subclass. Thanks for the nudge in the right direction! ;)